crest/Assets/Crest/Crest/Shaders/OceanInputs/Resources/InjectorSignedDistanceField.shader

```diff
@@ -0,0 +1,54 @@
+// Crest Ocean System
+
+// This file is subject to the MIT License as seen in the root of this folder structure (LICENSE)
+
+// Renders signed distance field without any depth
```

Comment:
Could you expand on this if possible? I'm looking at the code and not quite understanding how it works/what it does. I think the 'depth' value being unused is throwing me